cmd/thanos/query.go

```diff
@@ -71,7 +71,7 @@ func registerQuery(m map[string]setupFunc, app *kingpin.Application) {
 
 	instantDefaultMaxSourceResolution := modelDuration(cmd.Flag("query.instant.default.max_source_resolution", "default value for max_source_resolution for instant queries. If not set, defaults to 0s only taking raw resolution into account. 1h can be a good value if you use instant queries over time ranges that incorporate times outside of your raw-retention.").Default("0s").Hidden())
 
-	selectorLabels := cmd.Flag("selector-label", "Query selector labels that will be exposed in info endpoint (repeated).").
+	selectorLabels := cmd.Flag("selector-label", "Query selector labels are exposed at the info endpoint (repeated) and they comprise key/value pair of strings and we always try to have the subset of data that these labels point to, the store LabelSets and the selector-labels are merged and when there are no LabelSets from the store-proxy’s discovered stores we consider selector-labels as the LabelSets.").
```

Comment:
This makes sense, but it's the longest English sentence I have ever seen (: Do you mind splitting it into multiple ones? It's always easier to do so, it's easier to write some easy to read documentation. (: